service/source_validation.go

```diff
@@ -55,3 +55,15 @@ func ValidateSourceCreationRequest(dao dao.SourceDao, req *model.SourceCreateReq
 
 	return nil
 }
+func ValidateEditSourceNameRequest(dao dao.SourceDao, req *model.SourceEditRequest) error {
```

Comment:
Super small - but I like to have a space between function definitions and no space between the first stament here.